src/lib/y2storage/lvm_vg.rb

```diff
@@ -216,6 +216,68 @@ def potential_orphans
       lvm_pvs
     end
 
+    # Size of the physical volume that limits the maximum size of a new striped logical volume
+    #
+    # The maximum size of a striped logical volume is limited by the n-th biggest physical volume. For
+    # example, let's say we have a volume group with 3 physical volumes: pv1 (2 GiB), pv2 (50 GiB) and
+    # pv3 (1 GiB). For a striped volume with 2 stripes, the second biggest physical volume (pv1)
+    # restricts its maximum size. In this case, the maximum size would be:
+    # 2 (stripes) * 2 GiB (pv1 size) = 4 GiB.
+    # If the number of stripes is 3, then the maximum size is limited by the third biggest physical
+    # volume (pv3), so the maximum size would be: 3 (stripes) * 1 GiB (pv3 size) = 3 GiB.
+    #
+    # @param stripes [Integer] number of stripes. It must be bigger than 1
+    #
+    # @raise [RuntimeError] when the given number of stripes is incorrect (i.e., less than 1)
```

Comment:
Shouldn't this be an `ArgumentError`? To my taste `RuntimeError` is too generic.